src/notation/view/widgets/editstyle.cpp

```diff
@@ -48,6 +48,8 @@
 #include "ui/view/widgetstatestore.h"
 #include "ui/view/widgetutils.h"
 
+#include "internal/notationconfiguration.h"
```

Comment:
This should not be necessary. Note that `configuration()` returns the configuration as type `INotationConfiguration`, rather than `NotationConfiguration`, so you only need to #include `inotationconfiguration.h`, which is indeed already done in `editstyle.h`.